src/Integration/Binding/BindingWorkflow.cs

```diff
@@ -95,7 +95,7 @@ public List<NuGetPackageInfo> NuGetPackages
 
         public IProgressEvents Run()
         {
-            this.owner.UserNotification.HideNotification(NotificationIds.FailedToBindId);
+            this.host.ActiveSection?.UserNotifications?.HideNotification(NotificationIds.FailedToBindId);
```

Comment:
This is redundant since we're already clearing the notification in `BindingController.OnBindingStarted()`